Makefile

```diff
@@ -31,3 +31,6 @@ develop:
 
 wheels:
 	./scripts/build_wheels.sh
+
+test: 
```

Comment:
note that we use `tox` for testing, which tests against a number of python versions

Reply:
@arnthor89

Reply:
On it